.env.example

```diff
@@ -19,6 +19,7 @@ STORAGE_SECRET=
 
 # DEVELOPMENT VARIABLES
 TUNNEL_PORT=8080
+WORKER_PORT=4003
```

Comment:
When running app locally we need to have one port open for the worker health-check endpoint (But we wouldn't use it during for normal local devloop)

Reply:
can we maybe keep it consistent with the other port?  Maybe make it 8081?

Reply:
makes sense

Reply:
`bin/start-server-micros.sh`